README.md

```diff
@@ -1,6 +1,6 @@
 ![Next.js on Netlify Build Plugin](next-on-netlify.png)
 
-# Next.js Build Plugin
+# Essential Next.js Build Plugin
```

Comment:
@verythorough pinging for naming consistency check — this is correct, right?

Reply:
I think so, but @rstavchansky is the person driving our naming updates and can confirm (particularly about capitalization).

Reply:
Yes, @jlengstorf this is what we expect to see.